requirements.txt

```diff
@@ -23,3 +23,4 @@ securetar==2022.2.0
 sentry-sdk==1.9.8
 voluptuous==0.13.1
 dbus-next==0.2.3
+typing_extensions==4.3.0
```

Comment:
What is this needed for?

Reply:
it is needed to import `Required` in the block module which is needed in order to create a TypedDict with a dash in the key